ucsb-cs156-s24/team03-s24-4pm-8
# Jeffrey - HelpRequestCreatePage

Completed implementation of HelpRequestCreatePage and its relevant tests.
- Ensured 100% code coverage and 100% mutations killed


Storybook: [Here](https://ucsb-cs156-s24.github.io/team03-s24-4pm-8/prs/102/storybook/?path=/story/pages-helprequest-helprequestcreatepage--default)
![image](https://github.com/ucsb-cs156-s24/team03-s24-4pm-8/assets/61306390/ae85526d-b8ec-4568-9dcb-f456810cad2d)

To test:
1. Navigate to the page
2. Enter text for each of the inputs (if an input is invalid, it will tell you)
3. Click the 'submit' button
4. Ensure that a popup appears to signal that the request was valid

Closes #49
